specification/eventgrid/data-plane/Microsoft.AVS/preview/2023-09-01/examples/cloud-events-schema/script_execution_failed.json

```diff
@@ -0,0 +1,14 @@
+{
+  "source": "/subscriptions/{subscription_id}/resourceGroups/{resource_group_name}/providers/Microsoft.AVS/privateClouds/{sddc_name}",
+  "subject": "/scriptExecutions/{execution_name}",
+  "type": "Microsoft.AVS.ScriptExecutionFailed",
+  "time": "2023-12-09T13:50:19.9995668-08:00",
+  "id": "9b87886d-21a5-4af5-8e3e-10c4b8dac73b",
+  "data": {
+    "operationId": "fe1cb595-71c8-47d7-b505-60d39a44d53d",
+    "failure": "05/04/2023 03:15:40 Get-SpbmStoragePolicy SpbmStoragePolicy with name ' RAID-5 FTT-1' was not found using the specified filter(s).",
```

Comment:
Is it really "' RAID-5 FTT-1" or is that an extra leading space from you?

Reply:
This is an (almost verbatim) example of the error we've reported 2 weeks ago - we don't trim spaces apparently :)